jupyter_sphinx/css/jupyter-sphinx.css

```diff
@@ -57,7 +57,7 @@ div.jupyter_container {
 div.jupyter_container div.highlight {
   background-color: #f7f7f7; /* for haiku */
 }
-div.jupyter_container {
+div.jupyter_container * {
```

Comment:
Is this change accidental? I think it reverts a recent CSS bugfix.

Reply:
probably yeah, nice catch